DiagonalDifference.cs

```diff
@@ -0,0 +1,61 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    /*
+     * Complete the 'diagonalDifference' function below.
+     *
+     * The function is expected to return an INTEGER.
+     * The function accepts 2D_INTEGER_ARRAY arr as parameter.
+     */
+
+    public static int diagonalDifference(List<List<int>> arr)
+    {
+        int leftSum=0;
+        int rightSum=0;
+        int size=arr.Count;    //arr.Count gives us the size of the list(number of rows)
```

Comment:
No need to use an extra variable for the count. You can use arr.Count in the for loop itself

Reply:
Okay murthy,I actually declared it so that i could use it multiple times